src/test/java/org/fcrepo/client/integration/FcrepoTransactionIT.java

```diff
@@ -55,6 +58,20 @@ public void testTransactionCommit() throws Exception {
             location = response.getTransactionUri().orElseThrow(() -> new IllegalStateException("No tx found"));
         }
 
+        final var transactionalClient = FcrepoClient.client()
```

Comment:
I was thinking of storing the credentials in the client so that we could do this. Agreed that it makes sense to have the client create the transactional client, which is what I initially wanted.